test_absolute (__main__.TestExercice) ... ok
test_factorial (__main__.TestExercice) ... FAIL
test_prefixes (__main__.TestExercice) ... ok
test_summation (__main__.TestExercice) ... ok
test_verify_ages (__main__.TestExercice) ... FAIL

======================================================================
FAIL: test_factorial (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 62, in test_factorial
    'Mauvaise réponse'
AssertionError: Lists differ: [3628800, 3628800, 3628800, 3628800] != [1, 6, 40320, 3628800]

First differing element 0:
3628800
1

- [3628800, 3628800, 3628800, 3628800]
+ [1, 6, 40320, 3628800] : Mauvaise réponse

======================================================================
FAIL: test_verify_ages (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 77, in test_verify_ages
    'Mauvaise réponse'
AssertionError: Lists differ: [False, False, True, False, False, False, False, True, False, True, False] != [False, True, False, False, True, True, True, True, False]

First differing element 1:
False
True

First list contains 2 additional elements.
First extra element 9:
True

- [False, False, True, False, False, False, False, True, False, True, False]
+ [False, True, False, False, True, True, True, True, False] : Mauvaise réponse

----------------------------------------------------------------------
Ran 5 tests in 0.008s

FAILED (failures=2)
